### DISCUSSION TITLE
How to access tracker id of objects that crossed the line zone/ objects that are inside polygon zone

### DISCUSSION THREAD

**Discussion by abichoi** [roboflow/supervision#1433] [Q&A] [answered]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Is it possible to access the tracker id of objects that crossed the line zone or the id of objects that are inside the polygon zone?

**Comment by rolson24** [comment, marked as answer]
Hi @abichoi,
Yes you can access the tracker id of the objects that crossed the line zone like this:

```
zone = LineZone()
for frame in frames:
  # get the detections as normal
  detections_in_zone = detections[zone.trigger(detections=detections)]
  tracker_ids_in_zone = detections_in_zone.tracker_id
```
This will give you the detections in that zone at that frame. If you want to keep track of all of the detections that cross a line or enter a zone, you can just make a `set` with all of the tracker ids that gets added to every frame.

**Comment by SkalskiP** [comment]
Thanks for the help, @rolson24 🙏🏻 @abichoi. I'm converting this issue into a discussion if you have more questions.
